Volume/Views/WeeklyDebriefView.swift

```diff
@@ -30,21 +54,21 @@ struct WeeklyDebriefView: View {
                         .resizable()
                         .frame(width: 245, height: 75)
                         .padding(.top, 24)
-                    Text("Your weekly debrief, 5/3 - 5/9")
-                        .padding([.top, .bottom], 32)
+                    Text(getWeekString())
+                        .padding(.vertical, 32)
                         .font(.newYorkMedium(size: 16))
                     Divider()
                         .frame(width: 100)
                         .padding(.bottom, 48)
-                    VStack(spacing: 24, content: {
+                    VStack(spacing: 24) {
                         HStack {
                             Text("In the past week, you...")
                                 .font(.newYorkRegular(size: 16))
                             Spacer()
                         }
-                        StatisticView(image: "feed", leftText: "read", number: 10, rightText: "articles")
-                        StatisticView(image: "shout-out", leftText: "gave", number: 45, rightText: "shout-outs")
-                        StatisticView(image: "bookmark", leftText: "bookmarked", number: 10, rightText: "articles")
+                        StatisticView(image: "volume", leftText: "read", number: debriefDisplayInfo.numReadArticles, rightText: "articles")
```

Comment:
talk to han + justin about factoring out strings into a separate static file similar to what is done for images. 

this would be a separate PR